docs/src/components/Navigation.js

```diff
@@ -13,72 +14,50 @@ const isModifiedEvent = event =>
 
 const components = Object.keys(routes);
 
-export default function Navigation(props: Props) {
-  const { history } = props;
-  const links = components.map(ns => {
-    const href = history.createHref({ pathname: `/${ns}` });
-    const handleClick = ({ event }) => {
-      if (event.defaultPrevented) return;
-      if (isModifiedEvent(event) || !isLeftClickEvent(event)) return;
-      event.preventDefault();
-      history.push(`/${ns}`);
-    };
-    return (
-      <Text bold leading="tall" color="darkGray" size="lg">
-        <Link href={href} onClick={handleClick}>
-          {ns}
-        </Link>
-      </Text>
-    );
-  });
-  const options = [{ label: '-', value: '#' }].concat(
-    components.map(ns => ({
-      label: ns,
-      value: `/${ns}`,
-    }))
-  );
-  const m = window.location.hash.match(/#(\/[\w]+)/);
-
+export default function Navigation({ history }: Props) {
   return (
     <Box>
-      <Box mdDisplay="none" flex="grow">
-        <SelectList
-          id="nav"
-          onChange={({ value }) => history.push(value)}
-          options={options}
-          value={(m && m[1]) || '#'}
-        />
-      </Box>
-      <Box display="none" mdDisplay="flex" direction="column" flex="grow">
+      <Box
+        display="flex"
+        smDisplay="none"
+        overflow="scrollX"
+        paddingX={4}
+        paddingY={2}
+      >
         <Box
-          marginBottom={4}
           display="flex"
+          role="list"
           direction="row"
-          alignItems="center"
-          marginLeft={-1}
-          marginRight={-1}
+          marginStart={-2}
+          marginEnd={-2}
         >
-          <Box paddingX={1}>
-            <Icon
-              icon="pinterest"
-              color="red"
-              size={24}
-              accessibilityLabel="Pinterest Logo"
-            />
-          </Box>
-          <Box paddingX={1}>
-            <Heading size="xs">Gestalt</Heading>
-          </Box>
-        </Box>
-        <Box paddingY={2}>
-          <Link href="https://codesandbox.io/s/k5plvp9v8v" target="blank">
-            <Text>Try it in the sandbox</Text>
-          </Link>
+          {components.map((component, i) => (
+            <Box role="listitem" key={i} flex="none">
+              <NavLink to={`/${component}`}>
+                <Box padding={2}>{component}</Box>
+              </NavLink>
+            </Box>
+          ))}
         </Box>
-        <Box role="list">
-          {links.map((link, i) => (
-            <Box role="listitem" key={i}>
-              {link}
+      </Box>
+      <Box
+        display="none"
+        smDisplay="flex"
+        direction="column"
+        flex="grow"
+        role="list"
+        padding={4}
+        mdPadding={6}
+        lgPadding={8}
+      >
+        <Box marginStart={-2} marginEnd={-2}>
+          {components.map((component, i) => (
+            <Box role="listitem" key={i} flex="none">
```

Comment:
@chrislloyd do we need the `flex="none"` even though the container doesn't have `flex` set?